libs/application-generic/src/services/calculate-delay/compute-job-wait-duration.service.ts

```diff
@@ -82,6 +95,29 @@ export class ComputeJobWaitDurationService {
     return 0;
   }
 
+  private getUserRegularDelayMetadata(
+    chimeraResponse: IChimeraDigestResponse | IDelayOutput
+  ): { amount: number; unit: DigestUnitEnum } | undefined {
+    if (isUserRegular(chimeraResponse)) {
+      const unit = castToDigestUnitEnum(chimeraResponse?.unit);
+      const amount = chimeraResponse?.amount;
+
+      return { amount, unit };
+    }
+
+    return;
+  }
+
+  private getUserScheduledDelayMetadata(
+    chimeraResponse: IChimeraDigestResponse | IDelayOutput
+  ): { date: string } | undefined {
+    if (isUserScheduled(chimeraResponse)) {
+      return { date: chimeraResponse?.date };
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
🤓 **nitpick (non-blocking):** ‏ since `isUserScheduled()` has a type predicate, the `?` operator isn't needed in the scope of the if, because the chimeraResponse will be treated as the proper type